### PR TITLE
CAN-1366, 1414, 1454, 1463, 1326, 1466

### PR DESCRIPTION
**Pull request by ks-rahul**
1. CAN-1366 fixed. 
2. CAN-1414fixed.
3. CAN-1454 fixed.
4. CAN-1463 fixed.
5. CAN-1326 fixed.
6. CAN-1466 fixed.

### CODE REVIEW

File: src/components/ComponentPages/ForgotPassword/__test__/forgotOTP.test.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ describe("OTP page", () => {
     expect(heading).toBeInTheDocument();
     expect(
       screen.getByText(
-        "Note : OTP has been sent to your registered email address and phone number."
+        "Note : OTP has been sent to your registered email address."
```
Comment (akuks) [inline src/components/ComponentPages/ForgotPassword/__test__/forgotOTP.test.tsx]: Must be from a common message file

File: src/components/ComponentPages/SettingsUI/index.tsx [hunk 2]
```diff
@@ -75,8 +75,7 @@ const SettingsUI = () => {
           <TabPane tab="Direct Supported Camps" key="1">
             <div className={styles.text_checkbox_cont}>
               <div className={styles.notes}>
-                Note : To change support order of camp, drag & drop the camp box
-                on your choice position.
+                Note : To change support order of camp, drag & drop the camp box on your choice position.
```
Comment (akuks) [inline src/components/ComponentPages/SettingsUI/index.tsx]: must be from common message file

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/SettingsUI/index.tsx (hunk 2 above)
Comment (ks-rahul) [reply src/components/ComponentPages/SettingsUI/index.tsx]: moved

File: src/components/ComponentPages/ForgotPassword/__test__/forgotOTP.test.tsx (hunk 1 above)
Comment (ks-rahul) [reply src/components/ComponentPages/ForgotPassword/__test__/forgotOTP.test.tsx]: moved

**Review by akuks — APPROVED** [review]

**Pull request merged**